osm2gtfs/core/osm_connector.py

```diff
@@ -240,8 +249,11 @@ def get_stops(self, refresh=False):
             # valid stop_area candidade?
             if 'public_transport' in relation.tags:
                 if relation.tags["public_transport"] == "stop_area":
-                    self.stops["relation/" + str(relation.id)
-                               ] = self._build_stop_area(relation)
+                    try:
+                        self.stops["relation/" + str(relation.id)
+                                   ] = self._build_stop_area(relation)
+                    except RuntimeError:
+                        print('Cannot add stop area', relation.id)
```

Comment:
We should not catch `RuntimeError`s. Please use a different exception for this. You can even make your own.